docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -34,27 +34,70 @@ This guide assumes that you have:
 
 Teleport Cloud requires that plugins connect through the Proxy Service (`mytenant.teleport.sh:443`). Open Source and Enterprise installations can connect to the Auth Service (`auth.example.com:3025`) directly.
 
-### Create a user and role for access
+### Install the Teleport Slack Plugin
 
-(!docs/pages/includes/plugins/rbac.mdx!)
+We recommend installing the Teleport Plugins alongside the Teleport Proxy. This is an ideal
```

Comment:
I think we should move the following sentence into a `Notice`, which I'll include below: "We recommend installing the Teleport Plugins alongside the Teleport Proxy. This is an ideal location as plugins have a low memory footprint, and will require both public internet access and Teleport Auth access."

  ```markdown
  <Notice scope={["oss","enterprise"} type="tip">
  We recommend installing Teleport plugins on the same host as the Teleport
  Proxy Service. This is an ideal location as plugins have a low memory footprint
  and will require access to both the public internet and the Teleport Auth Service.
  </Notice>
  ```
  This will display this advice only for self-hosted users, since it doesn't apply to Teleport Cloud.